### ISSUE TITLE
about train code

### ISSUE THREAD

**Issue by onmygame** [CHELSEA234/HiFi_IFDL#13]
I found it seems there is mix between the train and test dataset
```
self.file_path = '/user/guoxia11/cvlshare/cvl-guoxia11/IMDL/REAL
 
self.file_path_fake = '/user/guoxia11/cvlshare/cvl-guoxia11/IMDL/FAKE
```
as the TrainData and ValData are using the same data root and the img_retreive function are the same

Also there is no model.train() called after the validation step the loss doesn't  reduce since the first epoch.

And a log of bugs in this code, I strongly suspect that there is data fraud in this paper

**Comment by GWZ313** [comment]
`I strongly agree with your opinion`
